### PR TITLE
rnafusion v1.0.2

### PR DESCRIPTION
**Pull request by matq007**
This was supposed to be `1.0.1-FIX`. Had to bump the version to `1.0.2`.

### CODE REVIEW

**Comment by apeltzer** [comment]
> This was supposed to be `1.0.1-FIX`. Had to bump the version to `1.0.2`.

I anyways like the numeric ones better ;-)

File: conf/awsbatch.config [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+ *  Nextflow config file for running on AWS batch
  * -------------------------------------------------
- * Imported under the 'awsbatch' Nextflow profile in nextflow.config
- * Uses docker for software depedencies automagically, so not specified here.
+ * Base config needed for running with -profile awsbatch
  */
+params {
+  config_profile_name = 'AWSBATCH'
+  config_profile_description = 'AWSBATCH Cloud Profile'
+  config_profile_contact = 'Alexander Peltzer (@apeltzer)'
```
Comment (MaxUlysse) [inline conf/awsbatch.config]: Why is @apeltzer the contact here?

File: environment.yml [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
   - anaconda::openjdk=8.0.152 # Needed for FastQC - conda build hangs without this
-  - bioconda::fastqc=0.11.8
   - bioconda::star=2.6.1b # update STAR-Fusion and Fusion-Inspector
-  - bioconda::multiqc=1.7
   - conda-forge::r-data.table=1.12.0
   - conda-forge::r-gplots=3.0.1.1
   - bioconda::bioconductor-edger=3.24.1
   - conda-forge::r-markdown=0.9
   - bioconda::fusion-report=1.0.0
+  - fastqc=0.11.8
```
Comment (MaxUlysse) [inline environment.yml]: Why did you remove the specification to the bioconda channel for fastQC and multiQC?

File: conf/awsbatch.config (hunk 1 above)
Comment (matq007) [reply conf/awsbatch.config]: This file was generated by the standard nf-core template 1.6. See the TEMPLATE branch [here](https://github.com/nf-core/rnafusion/blob/TEMPLATE/conf/awsbatch.config)

File: environment.yml (hunk 2 above)
Comment (matq007) [reply environment.yml]: Again, just following the [template](https://github.com/nf-core/rnafusion/blob/TEMPLATE/environment.yml).

File: conf/awsbatch.config (hunk 1 above)
Comment (apeltzer) [reply conf/awsbatch.config]: Yeah, the AWS Batch profile has me because I made the one being in the template and centrally taken care of...

File: environment.yml (hunk 2 above)
Comment (apeltzer) [reply environment.yml]: That is actually wrong in the template and should be there. Can you open an issue there to fix it so someone can take of it then?

Comment (matq007) [reply environment.yml]: Sure, I'll make a PR right away and change this in the pipeline as well!

Comment (matq007) [reply environment.yml]: Done

Comment (apeltzer) [reply environment.yml]: Let's keep it here for now until the discussion in the issue is resolved 👍 Don#t want to block it here because of just that, as it works fine and is just a "cosmetic" thing which doesn't break anything.

File: .github/bug_report.md [hunk 3]
```diff
@@ -1,29 +0,0 @@
-**Describe the bug**
```
Comment (apeltzer) [inline .github/bug_report.md]: Why is this gone?=

File: .github/feature_request.md [hunk 4]
```diff
@@ -1,16 +0,0 @@
-**Is your feature request related to a problem? Please describe.**
```
Comment (apeltzer) [inline .github/feature_request.md]: Should also stay 👍

File: tools/star-fusion/Dockerfile [hunk 5]
```diff
@@ -6,6 +6,7 @@ LABEL authors="rickard.hammaren@scilifelab.se, phil.ewels@scilifelab.se, martin.
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 ENV PATH /opt/conda/envs/star-fusion/bin:$PATH
+ENV TRINITY_HOME /opt/conda/opt/trinity-2.6.6
```
Comment (apeltzer) [inline tools/star-fusion/Dockerfile]: Do you need this?

File: .github/bug_report.md (hunk 3 above)
Comment (matq007) [reply .github/bug_report.md]: It has been moved to ISSUE_TEMPLATE.

File: .github/feature_request.md (hunk 4 above)
Comment (matq007) [reply .github/feature_request.md]: Again, has been moved to ISSUE_TEMPLATE. Should this is be in `.github` folder?

File: tools/star-fusion/Dockerfile (hunk 5 above)
Comment (matq007) [reply tools/star-fusion/Dockerfile]: The older version of STAR-Fusion required it. Maybe in the new version, I will drop it.

File: .github/feature_request.md (hunk 4 above)
Comment (apeltzer) [reply .github/feature_request.md]: Ah didn’t see it there - then it’s good and the other one too

File: tools/star-fusion/Dockerfile (hunk 5 above)
Comment (apeltzer) [reply tools/star-fusion/Dockerfile]: Okay 👌

**Comment by apeltzer** [comment]
Can’t accept on mobile, will do this tomorrow morning
